scripts/scripts.js

```diff
@@ -598,7 +597,8 @@ async function loadEager(doc) {
     decorateBreadcrumb(main);
     prepareSideNav(main);
     document.body.classList.add('appear');
-    await waitForLCP(LCP_BLOCKS);
+    const firstSection = main.querySelector('.section');
+    await loadSection(firstSection, waitForImage);
```

Comment:
I sometimes wonder if this shouldn't be even implementation details of the responsible block that should await internally before the end of the decoration.

Reply:
this would put extra burden on every block that could possibly host the LCP and worry about the distinction of being hosting the LCP and not...